src/features/licenses/components/LicenseForm/index.jsx

```diff
@@ -66,6 +66,10 @@ export const LicenseForm = ({
   const [optionSelection, setOptionSelection] = useState(created ? selectorOptions.courses : '');
 
   const showCatalogSelector = getConfig().SHOW_CATALOG_SELECTOR || false;
+  const isMultiCatalogSel = getConfig().MULTI_CATALOG_SELECTOR || false;
+
+  const disableLicenseNameIn = (created && !isMultiCatalogSel)
```

Comment:
Licence name In? I do not get it, I would suggest a refactor for this variable name